### ISSUE TITLE
Quick search through all friends

### ISSUE THREAD

**Issue by jcguu95** [mjkaufer/Messer#148]
## Summary

` search friend --regex ^[Aa]lexa*` gives

Alexandre Miljoe
Alexandra John
alexa

## Motivation

I have 1000+ friends. They might not use their real name on Facebook, or at least their not in canonical forms. Being able to search through friend list by filtering/regex.. much enhances the using experience.

**Comment by samueldy** [comment]
Agree, would be very nice to be able to do this within the app. Piping the output of `messer --command='contacts'` to grep would require a login each time it's done.

**Comment by jcguu95** [comment]
Or one that supports fuzzy "frecency" (frequency and recency) search.. that might be too much to ask :S
